tests/js/stay-alive-forever.js

```diff
@@ -0,0 +1,10 @@
+//# starling-not-a-test
+
+async function main() {
+  // Loop forever, but yield to the event loop so that we can get killed
+  // properly if the parent finishes.
+  while (true) {
+    print("child: uh uh uh uh, staying alive, staying alive");
+    await timeout(1000);
```

Comment:
It doesn't *really* matter, but can you make this a single very long timeout? That way we're yielding to the event loop exactly once, making this a slightly more targeted test. With the current setup, it'd be conceivable that we could only get killed upon yielding, not while we're waiting for a Future to be resolved.